core-bundle/src/Resources/config/listener.yml

```diff
@@ -273,3 +273,11 @@ services:
             - '@event_dispatcher'
         tags:
             - kernel.event_listener
+
+    Contao\CoreBundle\EventListener\DataContainer\DisableBundleConfiguredSettingsListener:
+        public: true
+        arguments:
+            - '@translator'
+            - '@contao.framework'
+        tags:
+            - { name: contao.callback, table: tl_settings, target: config.onload }
```

Comment:
You should use annotations instead (see https://github.com/contao/contao/pull/1762/files):

<img width="678" alt="" src="https://user-images.githubusercontent.com/1192057/85145372-c1fc0880-b24c-11ea-970e-664a34599c56.png">

Reply:
Okay, I changed the way of configuration. I honestly how the unit tests should look like to verify the correct implementation and the proposed PR doesn't test it at all. So I at least check if the annotation is there and the interface is implemented in 

https://github.com/contao/contao/pull/1382/commits/59e4b18039cd293c21662ff043569999ecd4507e#diff-5eec131ad59b19ad8080c1eb5e46225fR26-R42